docs/resources/samples/copy-tables-combine.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Merge multiple Excel tables into a single table'
+description: 'Learn how to use Office Scripts to combine data from multiple Excel tables into a single table.'
+ms.date: 03/30/2021
+localization_priority: Normal
+---
+
+# Copy Excel tables into a new table
+
+This sample combines data from multiple Excel tables into a single table that includes all the rows.
```

Comment:
Let's pull that bit about "same structure" into this description. I think it's important to be upfront that the tables need the same column headers. Plus, then it won't need to be in both bullets.